classes/class-list-table.php

```diff
@@ -464,10 +464,14 @@ function( $user_id ) {
 			);
 
 			if ( is_multisite() && is_super_admin() ) {
-				$super_admins = get_super_admins();
-				foreach ( $super_admins as $admin ) {
-					$users[] = get_user_by( 'login', $admin );
-				}
+				$super_admins = array_map(
+					function( $login ) {
+						$user = get_user_by( 'login', $login );
+						return new Author( $user->ID );
+					},
+					get_super_admins()
+				);
+				$users = array_unique( array_merge( $users, $super_admins ) );
```

Comment:
Array unique so we don't have duplicates if a Super Admin is also an Administrator for the site.